### DISCUSSION TITLE
Audio volume

### DISCUSSION THREAD

**Discussion by cYCL157** [rern/rAudio#108] [Fixed Issues]
After update this morning to 20210113 i have an issue with volume when the volume shows 0 on the dial and initially I had no volume out of the dac.  After playing, I found out that the mixer volume was set to 0 even though i had it set volume control to mixer device.  I can set the mixer volume to something other than 0 and get sound, but always at the same level.  I changed the volume control to mpd software which sets the mixer device volume to 0 and i get no sound, regardless of what I adjust that to.

It's been working great until the patch this morning.

I have a 3b with a hifiberry dac+, and have since the mixer device changes came out been setting volume control to mixer device, mixer device to digital and the mixer device volume gets set to 100.  with this i have been able to control the volume per the volume dial on the main page.

i did reboot after the update and then a few times and I was playing around to see if I could figure out what combination of settings might make it work to no differing result.

Thanks for any help and for something that i use day in and day out!

Cort

**Comment by rern** [comment]
Post result of:
`mpd-data.sh`

**Comment by MrsTrellis** [comment]
I see exactly the same symptoms as the OP (I have 2 * Pi 4B with HiFiBerry DAC+).  Oddly I thought it was working fine yesterday after the 20210113 installation, but this behaviour started today.  Perhaps I am mistaken on that point!

mcfly:~  #  mpd-data.sh
{ "devices" : [{ "aplayname" : "hifiberry-dacplus" , "card" : 0 , "device" : 0 , "dop" : 0 , "hw" : "hw:0,0" , "hwmixer" : "Digital" , "mixers" : 1 , "mixerdevices" : ["Digital"] , "mixermanual" : false , "mixertype" : "hardware" , "name" : "HiFiBerry DAC+" }] , "active" : true , "asoundcard" : 0 , "audioaplayname" : "hifiberry-dacplus" , "audiooutput" : "HiFiBerry DAC+" , "autoupdate" : false , "buffer" : false , "bufferval" : false , "bufferoutput" : false , "bufferoutputval" : false , "counts" : { "album": 666, "albumartist": 163, "artist": 557, "composer": 403, "date": 69, "genre": 71, "nas": 0, "sd": 148, "usb": 0, "song": 9397, "webradio": 14 } , "crossfade" : false , "crossfadeval" : false , "custom" : false , "ffmpeg" : false , "normalization" : false , "reboot" : "" , "replaygain" : false , "replaygainval" : "" , "soxr" : false , "soxrval" : "" , "version" : "0.22.3-2" }

**Reply by rern** [reply]
Please get another one:
`status.sh`

**Reply by MrsTrellis** [reply]
mcfly:~  #  status.sh
{ "player" : "mpd" , "btclient" : false , "consume" : false , "control" : "Analogue" , "counts" : { "album": 666, "albumartist": 163, "artist": 557, "composer": 403, "date": 69, "genre": 71, "nas": 0, "sd": 148, "usb": 0, "song": 9397, "webradio": 14 } , "lcd" : false , "librandom" : false , "playlistlength" : 1 , "playlists" : 0 , "relays" : false , "relayson" : false , "updateaddons" : false , "volume" : 100 , "volumemute" : 0 , "repeat" : false , "random" : false , "single" : false , "elapsed" : 6 , "file" : "http://media-ice.musicradio.com:80/LBCUK" , "song" : 0 , "state" : "play" , "timestamp" : 1610622625934 , "updating_db" : false , "volumemute" : 0 , "Album" : "LBC (National)" , "Artist" : "" , "Name" : "LBC UK" , "Time" : false , "Title" : "" , "webradio" : true , "sampling" : "1/1 &bull; 16 bit 44.1 kHz 56 kbit/s" , "coverartradio" : "/data/webradiosimg/http:||media-ice.musicradio.com:80|LBCUK.1610622626.jpg" , "coverart" : ""}

**Reply by rern** [reply]
Try:
`sed -i "s/pvolume/'volume.*pswitch'/" /srv/http/bash/cmd.sh`
Refresh Playback page then try the volume again.

**Reply by MrsTrellis** [reply]
That's fixed it :-) :-) Thank you!!
